models/marts/core/fct_claims.sql

```diff
@@ -4,32 +4,50 @@ with claims as (
 
 ),
 
+conditions as (
+
+    select * from {{ ref('stg_conditions') }}
+
+),
+
+procedures as (
+
+    select * from {{ ref('stg_procedures') }}
+
+),
+
 final as (
 
     select
         -- primary key
-        claim_id,
+        claims.claim_id,
 
         -- foreign keys
-        organization_id,
-        encounter_id,
-        condition_id,
-        medication_request_id,
-        patient_id,
-        procedure_id,
+        claims.organization_id,
+        claims.encounter_id,
+        claims.condition_id,
+        claims.medication_request_id,
+        claims.patient_id,
+        claims.procedure_id,
 
         -- dimensions
-        claim_status,
+        conditions.condition_type,
+        procedures.procedure_code,
+        claims.claim_status,
 
         -- measures
-        claim_total_value,
-        timestamp_diff(billable_period_ended_at,billable_period_started_at, minute) as billable_period_duration_minutes,
+       claims.claim_total_value,
```

Comment:
```suggestion
        claims.claim_total_value,
```
Indent in the web view looks off